examples/openstack-ansibleee-play-runner-extra-vars.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: ansibleee.openstack.org/v1alpha1
+kind: OpenStackAnsibleEE
+metadata:
+  name: ansibleee-play
+  namespace: openstack
+spec:
+  image: quay.io/openstack-k8s-operators/openstack-ansibleee-runner:latest
+  play: |
+    - name: Print hello world
+      hosts: all
+      tasks:
+      - name: Using debug statement
+        ansible.builtin.debug:
+          msg: "{{ secretmsg | default('Hello world!'') }}"
+  extraMounts:
+    - volumes:
+      - name: inventory
+        configMap:
+          name: inventory-configmap
+      - name: sshkey
+        secret:
+          secretName: ssh-key-secret
+          items:
+          - key: ssh-privatekey
+            path: ssh_key
+      mounts:
+      - name: inventory
+        mountPath: /runner/inventory/inventory.yaml
+        subPath: inventory.yaml
+      - name: sshkey
+        mountPath: /runner/env/ssh_key
+        subPath: ssh_key
+  extraVars:
+    secretmsg: "Hello Mars!"
```

Comment:
This is a more convenient way to pass `extraVars`. Thank you Jiri.

Reply:
You are welcome :)